### PR TITLE
Typescript for Progress stories

### PR DESCRIPTION
**Pull request by wenche**
Part of #735

### CODE REVIEW

**Comment by wenche** [comment]
I don't think it's possible to avoid the uncontrolled vs controlled component react error message in dev console when we use the control option for value 🤷‍♀️ 
Skjermbilde 2020-10-28 kl  10 25 53

File: apps/storybook-react/stories/Progress Indicators/hooks/useProgress.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
```
Comment (mimarz) [inline apps/storybook-react/stories/Progress Indicators/hooks/useProgress.tsx:1]: This should maybe be `.ts`?

Comment (wenche) [reply apps/storybook-react/stories/Progress Indicators/hooks/useProgress.tsx:1]: Good catch! Not used to writing .tsx 👍

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**